notebooks/iceflow/3_dataviz.ipynb

```diff
@@ -0,0 +1,240 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib widget\n",
+    "import vaex\n",
+    "from ipywidgets import widgets\n",
+    "import matplotlib.pyplot as plt\n",
+    "import cartopy.crs as ccrs\n",
+    "\n",
+    "plt.figure(figsize=(10,8), dpi= 90)\n",
+    "ax = plt.axes(projection=ccrs.SouthPolarStereo(central_longitude=0)) \n",
+    "ax.coastlines(resolution='50m', color='black', linewidth=1)\n",
+    "ax.set_extent([-180, 180, -65, -90], ccrs.PlateCarree())\n",
+    "plt.scatter(df_decimated[df_key['longitude']].values,\n",
+    "            df_decimated[df_key['latitude']].values,\n",
+    "            c=df_decimated[df_key['elevation']].values,\n",
+    "            cmap='viridis',\n",
+    "            vmin=100,vmax=1000,\n",
+    "            transform=ccrs.PlateCarree())\n",
+    "plt.colorbar(label='elevation', shrink=0.5, extend='both')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## \"Flying\" with the sensor to get a closer look on the data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib widget\n",
+    "from ipywidgets import widgets\n",
+    "from ipywidgets import interact, interactive, fixed\n",
+    "from mpl_toolkits.mplot3d import Axes3D\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "\n",
+    "fig = plt.figure(figsize=(10,6))\n",
+    "ax = fig.add_subplot(111, projection='3d')\n",
+    "ax.view_init(70, 70)\n",
+    "\n",
+    "#If the data granule(s) is big enough (1+GB), use the decimated dataframe.\n",
+    "# df = df_decimated\n",
+    "\n",
+    "def plot_func(alontrack):\n",
+    "    step = 5000 # same as density\n",
+    "    m = int(alontrack * step)\n",
+    "    ax.clear()\n",
+    "    ax.scatter(df[df_key['longitude']].values[m:m+step],\n",
+    "               df[df_key['latitude']].values[m:m+step],\n",
+    "               df[df_key['elevation']].values[m:m+step],\n",
+    "               c=df[df_key['elevation']].values[m:m+step],\n",
+    "               cmap='viridis', s=1)\n",
+    "    ax.axis('tight')\n",
+    "    \n",
+    "    \n",
+    "\n",
+    "interact(plot_func, alontrack = widgets.FloatSlider(value=0,\n",
+    "                                                    description='Along Track Steps',\n",
+    "                                                    min=0,\n",
+    "                                                    max=90,\n",
+    "                                                    step=0.3,\n",
+    "                                                    layout={'width': '100%'}))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Time Series\n",
+    "\n",
+    "After harmonizing data from Pre-IB, IS1, IceBridge, IS2 we should be able to grid and or concatenate the dataframes and start building real analyses.\n",
+    "\n",
+    "Ideas: \n",
```

Comment:
Is this still a todo for the notebook? These look like internal notes, not something we should publish.